src/collapse/dtrace.rs

```diff
@@ -27,7 +32,7 @@ pub struct Folder {
     /// Number of times each call stack has been seen.
     occurrences: HashMap<String, usize>,
 
-    opt: Options,
+    _opt: Options, // to keep the interface consistant
```

Comment:
I think we probably want to keep the `includeoffset` option that upstream has. We don't have to expose it as a CLI argument, but internally it seems like a good idea!

Reply:
👍 done